bokehjs/src/lib/models/layouts/layout_dom.ts

```diff
@@ -242,12 +242,21 @@ export abstract class LayoutDOMView extends DOMView {
     super.connect_signals()
 
     if (this.is_root)
-      window.addEventListener("resize", () => this.resize())
+      window.addEventListener("resize", this)
 
     // XXX: this.connect(this.model.change, () => this.layout())
     this.connect(this.model.properties.sizing_mode.change, () => this.layout())
   }
 
+  handleEvent(): void {
+    this.resize()
+  }
+
+  disconnect_signals(): void {
+    super.disconnect_signals()
+    window.removeEventListener("resize", this)
```

Comment:
It doesn't matter here, but this should be done in reverse.

Reply:
You mean the super call should go at the end of the method? I checked and there should be no impact to the ordering indeed, so I followed the pattern in connect_signals with super call at the top. But thinking about it again, it seems logical to mirror it.